### PR TITLE
 Typescript. Use Karakum for conversion.

### PR DESCRIPTION
**Pull request by sgrishchenko**

### CODE REVIEW

File: kotlin-typescript/src/jsMain/generated/typescript/ClassificationType.kt [hunk 1]
```diff
@@ -1,84 +1,35 @@
 // Automatically generated - do not modify!
 
-package typescript
+@file:JsModule("typescript")
 
-import seskar.js.JsIntValue
+package typescript
 
 sealed external interface ClassificationType {
```
Comment (turansky) [inline kotlin-typescript/src/jsMain/generated/typescript/ClassificationType.kt:7]: Runtime?

File: kotlin-typescript/src/jsMain/generated/typescript/Classifications.kt [hunk 2]
```diff
@@ -2,9 +2,7 @@
 
 package typescript
 
-import js.array.ReadonlyArray
```
Comment (turansky) [inline kotlin-typescript/src/jsMain/generated/typescript/Classifications.kt:5]: Could we restore imports for `ReadonlyArray`?

File: kotlin-typescript/src/jsMain/generated/typescript/CompilerHost.kt [hunk 3]
```diff
@@ -27,34 +26,24 @@ sealed external interface CompilerHost : ModuleResolutionHost {
...
-        typeDirectiveReferences: ReadonlyArray<Any /* T (FileReference | string) */>,
-        containingFile: String,
-        redirectedReference: ResolvedProjectReference?,
-        options: CompilerOptions,
-        containingSourceFile: SourceFile?,
-        reusedNames: ReadonlyArray<Any /* T (FileReference | string) */>?,
-    ) -> ReadonlyArray<ResolvedTypeReferenceDirectiveWithFailedLookupLocations>)?
+
+    /**
+     * @deprecated supply resolveTypeReferenceDirectiveReferences instead for resolution that can handle newer resolution modes like nodenext
```
Comment (turansky) [inline kotlin-typescript/src/jsMain/generated/typescript/CompilerHost.kt:40]: Deprecated = not generated?

**Review by turansky — CHANGES_REQUESTED** [review]

File: kotlin-typescript/src/jsMain/generated/typescript/Classifications.kt (hunk 2 above)
Comment (sgrishchenko) [reply kotlin-typescript/src/jsMain/generated/typescript/Classifications.kt:5]: Import hosting is not safe without AST analysis and  Karakum is not good tool to provide AST modifications for Kotlin code.

File: kotlin-typescript/src/jsMain/generated/typescript/ClassificationType.kt (hunk 1 above)
Comment (sgrishchenko) [reply kotlin-typescript/src/jsMain/generated/typescript/ClassificationType.kt:7]: TS enum is runtime entity

**Pull request merged**